corehq/apps/api/tasks.py

```diff
@@ -0,0 +1,10 @@
+import time
+from celery.schedules import crontab
+from celery.task.base import periodic_task
+from tastypie.models import ApiAccess
+
+
+@periodic_task(run_every=crontab(minute=0, hour=0), queue='background_queue')
+def clean_api_access():
+    accessed = int(time.time()) - 30 * 24 * 3600  # only keep last 30 days
+    ApiAccess.objects.filter(accessed__lt=accessed).delete()
```

Comment:
With no index this might be a painful delete: https://github.com/django-tastypie/django-tastypie/blob/master/tastypie/models.py#L20